.git-blame-ignore-revs

```diff
@@ -1,8 +1,20 @@
-# Let pre-commit format the code
-a3e059110584c0ed44a707e855df6adea5f70c78
+# [clang-tidy] Automatic fixes with a little handwork
+cdacc58428fa9063ce4b9f5f6b8a1d297720ae09
```

Comment:
This is not just formatting. These are actual code changes. Better not ignore those.